### PR TITLE
Setup the mobile portfolio toolbar and the header .

### PR DESCRIPTION
**Pull request by medsonmoombe**
**In this project I did the following :**

  
- Add images folder
- Add css file
 -  Add html file
 - Add gitignore file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <title>My portfolio</title>
+</head>
+<body>
+       <nav class="header-toolbar">
+           <h1>emmanuel moombe</h1> 
+           <i class="fa-solid fa-bars"></i>
+       </nav>
```
Comment (mahbubul14) [inline index.html]: - [ ] You should use the `header` tag here. Please wrap the `nav` tag with the `header` tag. :writing_hand:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                        <ul class="icons">
+                         <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
+                         <li><a href="#"><i class="fa-brands fa-linkedin"></i></a></li>
+                         <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li>
+                         <li><a href="#"><i class="fa-brands fa-twitter-square"></i></a></li>
+                         <li><a href="#"><i class="fa fa-medium"></i></a></li>
+                      </ul>
+                  </section>
+         </div>
+        </header>
```
Comment (mahbubul14) [inline index.html]: - [ ] Kindly change the `header` tag to the `section' tag for semantic purposes. :writing_hand:

File: index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+           <h1>emmanuel moombe</h1> 
+           <i class="fa-solid fa-bars"></i>
+       </nav>
+       <header class="container">
+          <div class="row">
+                <h2>Hey There.
+                  I’m Emmanuel</h2>
+                <h3>I’am a Software Developer</h3>
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                  don’t hesitate to contact me.</p>
```
Comment (mahbubul14) [inline index.html]: **Currently, your design is not matched with Figma's design. Please work on that. :writing_hand:** 

- [ ] The `h2` tag should be `h1` tag. The text will be bigger to match the Figma design.

- [ ] Header and content space should be reduced a bit.

- [ ] You should reduce the Left and right side space for the content to match the design.


Your design             |  Figma Design
:-------------------------:|:-------------------------:
y1 | f1

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @medsonmoombe ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Check the comments under the review for more suggestions. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

- [ ] For readability, please add two spaces of indentation. Do not use the tab key. You did it throughout your code in the `HTML` file. Please fix it. [For the HTML & CSS best practices ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)we should follow the rules.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by medsonmoombe** [comment]
## did as requested by the code reviewer
 - change the header to a section tag 
 - Add the header tag and enclose the nav  
 - Change the margins and padding to center the content as the figma template
 -

**Review by BenMukebo — APPROVED** [review]
## Hi *** @medsonmoombe, *** 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations, I really appreciate your work oh art, keep the great job Medson 💪🏻 ! 🎉  🎉 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**